README.md

```diff
@@ -401,6 +401,29 @@ private void OnMessage(DetailedResponse<MessageResponse> response, IBMError erro
 }
 ```
 
+## Transaction IDs
+Every SDK call will return a response which will contain a transaction ID, accessible via the `x-global-transaction-id` header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
+
+```cs
+AssistantService service = new AssistantService("{version-date}");
+
+public IEnumerator ExampleGetTransactionId()
+{
+    WorkspaceCollection listWorkspacesResponse = null;
+    service.ListWorkspaces(
+        callback: (DetailedResponse<WorkspaceCollection> response, IBMError error) =>
+        {
+            var globalTransactionId = response.Headers["x-global-transaction-id"];
+            Log.Debug("AssistantServiceV1", "Global transaction id: {0}", globalTransactionId);
+            listWorkspacesResponse = response.Result;
+        }
+    );
+
+    while (listWorkspacesResponse == null)
+        yield return null;
+}
+```
+
```

Comment:
Do you get the transaction id if you have an error as well? ie if you call GetWorkspace with a bad workspaceId?

Reply:
updated.